tests/network/migration/test_masquerade_connectivity_after_migration.py

```diff
@@ -53,11 +54,22 @@ def running_vm_for_migration(
     ) as vm:
         vm.start(wait=True)
         vm.wait_for_agent_connected()
+
+        static_vm_ip = running_vm_static.vmi.interfaces[0]["ipAddress"]
+
+        vm_console_run_commands(
```

Comment:
The rational for this part is clear from the context of this PR, but once merged - it won't be clear what does a connectivity action have to do in a fixture which is supposed to spin-up a VM and make sure it's up (and the fixture name also implies that).
So please add a short comment explaining why this ping is done.
Thank you